include/cista/cista_exception.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <exception>
+#include "cista/exception.h"
```

Comment:
```suggestion

#include "cista/exception.h"
```

please separate std includes and non-std includes